bip/t3dn_bip/utils.py

```diff
@@ -127,16 +153,15 @@ def _should_resize(size: tuple, max_size: tuple) -> bool:
     return False
 
 
-def _resize_image(image: Image.Image, max_size: tuple) -> Image.Image:
+def _resize_image(image: 'Image.Image', max_size: tuple) -> 'Image.Image':
```

Comment:
Just curious, because I dont know much about Python type hints: why do we need to quote this type?

Reply:
It's either that or `from __future__ import annotations` because Image.Image does not exist if Pillow was not successfully imported.